FAQ.md

```diff
@@ -3,502 +3,565 @@ title: TiDB FAQ
+理论上误差越小越好，切换 leader 的时候如果时钟回退，就会卡住直到追上之前的 leader。这个容忍是业务上的，PD 多长的误差都能容忍。 但是误差越大，主从切换的时候，停止服务的时间越长。
 
-#### TiDB 是否可以使用 SAS/SATA 盘或者进行 SSD/SAS 混合部署？
-不可以使用，TiDB 在进行 OLTP 场景中，数据访问和操作需要高 IO 磁盘的支持，TiDB 作为强一致的分布式数据库，存在一定的写放大，如副本复制、存储底层 Compaction，因此，TiDB 部署的最佳实践中推荐用户使用 NVMe SSD 磁盘作为数据存储磁盘。另外，TiKV 与 PD 不能混合部署。
+### 3.2.4 Client 连接是如何寻找 PD 的？
 
-#### 有没有图形化部署 TiDB 的工具？
-暂时没有。
+Client 连接只能通过 TiDB 访问集群，TiDB 负责连接 PD 与 TiKV，PD 与 TiKV 对 Client 透明。当 TiDB 连接任意一台 PD 的时候，PD 会告知 TiDB 当前的 leader 是谁，如果此台 PD 不是 leader ，TiDB 将会重新连接至 leader PD。
+
+### 3.2.5 PD 参数中 leader-schedule-limit 和 region-schedule-limit 调度有什么区别？
+
+- leader-schedule-limit 调度是用来均衡不同 TiKV 的 leader 数，影响处理查询的负载。
+- region-schedule-limit 调度是均衡不同 TiKV 的副本数，影响不同节点的数据量。
+
+### 3.2.6 每个 region 的 replica 数量可配置吗？调整的方法是？
+
+可以，目前只能调整全局的 replica 数量。首次启动时 PD 会读配置文件（conf/pd.yml），使用其中的 max-replicas 配置，之后修改需要使用 pd-ctl 配置命令 `config set max-replicas $num`，配置后可通过 `config show all` 来查看已生效的配置。调整的时候，不会影响业务，会在后台添加，注意总 TiKV 实例数总是要大于等于设置的副本数，例如 3 副本需要至少 3 个 TiKV。增加副本数量之前需要预估额外的存储需求。pd-ctl 的详细用法可参考 [PD Control 使用说明](https://pingcap.com/docs-cn/FAQ/tools/pd-control.md)。
+
+### 3.2.7 缺少命令行集群管理工具，整个集群的健康度当前是否正常，不好确认？
+
+可以通过 pd-ctl 等工具来判断集群大概的状态，详细的集群状态还是需要通过监控来确认。
+
+### 3.2.8 集群下线节点后，怎么删除老集群节点监控信息？
+
+下线节点一般指 TiKV 节点通过 pd-ctl 或者监控判断节点是否下线完成。节点下线完成后，手动停止下线节点上相关的服务。从 Prometheus 配置文件中删除对应节点的 node_exporter 信息。从 Ansible inventory.ini 中删除对应节点的信息。
+
+## 3.3 TiDB server 管理
+
+### 3.3.1 TiDB 的 lease 参数应该如何设置？
+
+启动 TiDB Server 时，需要通过命令行参数设置 lease 参数（--lease=60），其值会影响 DDL 的速度（只会影响当前执行 DDL 的 session，其他的 session 不会受影响）。在测试阶段，lease 的值可以设为 1s，加快测试进度；在生产环境下，我们推荐这个值设为分钟级（一般可以设为 60），这样可以保证 DDL 操作的安全。
+
+### 3.3.2 TiDB 是否支持其他存储引擎？
+
+是的，除了 TiKV 之外，TiDB 还支持一些流行的单机存储引擎，比如 Goleveldb/Rocksdb/Boltdb 等。如果一个存储引擎是支持事务的 KV 引擎，并且能提供一个满足 TiDB 接口要求的 Client，即可接入 TiDB。
+
+### 3.3.3 为什么有的时候执行 DDL 会很慢？
+
+可能原因如下：
+
+- 多个 DDL 语句一起执行的时候，后面的几个 DDL 语句会比较慢。原因是当前 TiDB 集群中 DDL 操作是串行执行的。
+- 在正常集群启动后，第一个 DDL 操作的执行时间可能会比较久，一般在 30s 左右，这个原因是刚启动时 TiDB 在竞选处理 DDL 的 leader。
+- 在滚动升级或者停机升级时，由于停机顺序（先停 PD 再停 TiDB）或者用 `kill -9` 指令停 TiDB 导致 TiDB 没有及时清理注册数据，那么会影响 TiDB 启动后 10min 内的 DDL 语句处理时间。这段时间内运行 DDL 语句时，每个 DDL 状态变化都需要等待 2 * lease（默认 lease = 10s）。
+- 当集群中某个 TiDB 与 PD 之间发生通讯问题，即 TiDB 不能从 PD 及时获取或更新版本信息，那么这时候 DDL 操作的每个状态处理需要等待 2 * lease。
 
-#### 不想用 Ansible 部署 TiDB，能否支持其他部署的方法？
-除 Ansible 之外，还支持 Docker 容器化部署，详细可在[官网](https://pingcap.com/docs-cn/op-guide/docker-deployment/)进行检索。官方更推荐 Ansible 方式，可以对集群主机做出相关优化以及检测，可以自动生成启动、停止、升级等脚本方便后期管理，同时部署了相关监控系统，在测试与调优以及排查问题时非常直观。
+### 3.3.4 TiDB 可以使用 S3 作为后端存储吗？
 
-#### 可否在单机上安装开发环境？
-可以，使用 `docker-compose` 在本地一键拉起一个集群，包括集群监控，还可以根据需求自定义各个组件的软件版本和实例个数，以及自定义配置文件。详细可参考[官方文档](https://github.com/pingcap/tidb-docker-compose)。
+不可以，目前 TiDB 只支持分布式存储引擎和 Goleveldb/Rocksdb/Boltdb 引擎；
+
+### 3.3.5 Infomation_schema 能否支持更多真实信息？
+
+Infomation_schema 库里面的表主要是为了兼容 MySQL 而存在，有些第三方软件会查询里面的信息。在目前 TiDB 的实现中，里面大部分只是一些空表。后续随着 TiDB 的升级，会提供更多的参数信息。当前 TiDB 支持的：Infomation\_schema 请参考 [TiDB 系统数据库说明文档](https://pingcap.com/docs-cn/SQL/system-database/)。
```

Comment:
Fix the link to:
https://pingcap.com/docs-cn/sql/system-database/

Pls check whether the link is valid before adding it to the document.